include/ofi_mr.h

```diff
@@ -283,6 +283,17 @@ struct ofi_mr {
 	void *hmem_data;
 };
 
+static inline bool ofi_mr_all_host(struct ofi_mr **mr, size_t count)
+{
+	int i;
+
+	for (i = 0; i < count; i++) {
+		if (mr[i] && mr[i] != FI_HMEM_SYSTEM)
```

Comment:
mr[i].iface ?

Does this build cleanly?

Reply:
whoops. it does build cleanly but yup that's a mistake! will fix. Sorry.